domains/windowsed.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "windowsed1225",
+           "email": "me@windowsed.me",
+           "discord": "906415095473655810",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.Wu2s52VXs8ZJdqRj-2BQR7uW8Y5PTkyDREXRM18p7MHsubvopdIV4SfcNaXFcYvjiyJBUeq-txAqRnlebZU_Z1RgeoN0raa50fEo1jyiXEngDzvnDcnqlWUeI49eXP-2JSbUEXm3Y_sX2tmjZvlyUg0gN1rNa7-p13d2WJB1yWG-DEQK680lyJ63q_3ImzjkzzwJaiNtqInvbro6IOI0zOGREvoycbSFB7dICeTodq5BNYm5coBrY1g_QxmhL5x6uVovN64SJssy1MAKhr8i2iR3KhfJkHWFIiaJviGfJqJic4w3TKIFPlvelclyqMDFatFdaR_cNSEQZNZrQLAafQ.MDwq2TyPR64xGApOjOEllQ.7sfYfncrWm11i5oFs_I5-kNFptjdv6vuqksAgUwBWJpSpY3vytpHY9Sm6vRT1E0K1A7nMlTcPFa5mYm-EuxkqigLAis2uqK3E0ydYXZ75kABX0JT7Qg8r3U10XGpjeKL0-5WnKRvytun9O7XJGuLPQ.-Q_dogK2zHsV1DuGhhaF-A"
+        },
+    
+        "record": {
+            "TXT": "vc-domain-verify=windowsed.is-a.dev,e5344d1f26e8286585de"
```

Comment:
This is a verification record that should be placed under a _vercel domain file.

Reply:
i resolve that.